service/src/test/java/datawave/microservice/audit/replay/AuditReplayTest.java

```diff
@@ -66,14 +66,15 @@
 import static datawave.webservice.common.audit.AuditParameters.QUERY_SECURITY_MARKING_COLVIZ;
 import static datawave.webservice.common.audit.AuditParameters.QUERY_STRING;
 import static datawave.webservice.common.audit.AuditParameters.USER_DN;
-import static org.junit.Assert.assertNotNull;
 
-@RunWith(SpringRunner.class)
+@ExtendWith(SpringExtension.class)
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 @ContextConfiguration(classes = AuditReplayTest.AuditReplayTestConfiguration.class)
 @ActiveProfiles({"AuditReplayTest", "replay-config"})
 public class AuditReplayTest {
     
+    private static final long TEST_TIMEOUT_MILLIS = TimeUnit.SECONDS.toMillis(100);
```

Comment:
It is pretty long.  That's the timeout for a single operation including retries.  The audit replay test is timing dependent, and we have run into issues before with shorter timeouts when multiple tests are running in parallel during a build.  Eventually, it might be good to see if we can refactor the test in such a way that this can be reduced.